.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,24 @@
-### Context
+## Ticket and context
 
-### Changes proposed in this pull request
+Ticket:
 
-### Guidance to review
+We want to xxx because yyy.
 
-### Testing
+## Code review
+
+### What does this PR change in code?
+1. Change xxx in yyy
+
+### Is there anything weird that code reviewer should know?
+Nope, everything is straightforward.
+
+### Review Checks
+- [ ] All model methods are unit-tested via model specs
+- [ ] All controller actions are unit-tested via request specs
+- [ ] All pages have automated accessibility checks via cypress
+- [ ] All pages have visual tests via cypress + percy
+- [ ] All forms have an `id` attribute on them
+
+## Product review
+
+### How can someone see it it review app?
```

Comment:
We don't know the review app URL at the point the PR is open. Maybe some default text saying to click the link posted by the bot in the comment below?